### PR TITLE
Sulawesi combined

### PR DESCRIPTION
**Pull request by smitpau4**

### CODE REVIEW

**Review by hjrrockies — COMMENTED** [review]
I haven't kept up with the code changes very well since I graduated, but the original (and strongly recommended) code design is that nothing that is scenario-specific should be put in the core tsunamibayes classes. So these changes (which appear to date back a while) that make GeoClawForwardModel specific to the Sulawesi 1820 scenario are misplaced, because it will break other scenarios that depended on the old classes.

My strong recommendation is that, when adjustments are needed for a specific scenario, a new class inheriting from GeoClawForwardModel (or BaseForwardModel) should be created and located within the Sulawesi scenario directory.

**Comment by hjrrockies** [comment]
To elaborate a bit: the idea is that any changes to the core tsunamibayes module should not _ever_ break the ability to run previous scenarios. The code, before the 2020 redesign, suffered from major incompatibility issues due to some ad-hoc choices made in the design process. While updates to the core module are obviously going to be needed, they should always be done in a way that preserves the "scenario-invariant" design principle.

**Comment by jpw37** [comment]
Hayden, you have the right idea here, but we have found that although we
tried to keep the scenarios separate from the tsunamibayes changes it
wasn't completely possible.  This particular merge should have been done on
a different branch until we can figure out how to modify the main code in
tsunamibayes such that the changes in the scenarios can capture all of the
unique settings we are working with.

Basically we need to revert back to an earlier version prior to this most
recent merge and then try to move forward from there.  I'm occupied today
and this weekend, but you definitely have the right idea on this.

Jared Whitehead
Associate Professor and Graduate Coordinator,
Mathematics Department
Brigham Young University
350 TMCB, Provo, UT, USA
http://tsunami.byu.edu
http://www.mormon.org/me/15TK
